Documentation/ceph-cluster-crd.md

```diff
@@ -74,7 +72,7 @@ For more details on the mons and when to choose a number other than `3`, see the
 - `storage`: Storage selection and configuration that will be used across the cluster.  Note that these settings can be overridden for specific nodes.
   - `useAllNodes`: `true` or `false`, indicating if all nodes in the cluster should be used for storage according to the cluster level storage selection and configuration values.
   If individual nodes are specified under the `nodes` field below, then `useAllNodes` must be set to `false`.
-  - `nodes`: Names of individual nodes in the cluster that should have their storage included in accordance with either the cluster level configuration specified above or any node specific overrides described in the next section below.
+  - `nodes`: Names of individual nodes in the cluster that should have their storage included in accordance with either the cluster level configuration specified above or any node specific overrides described in the next section below. See [node settings](#node-settings) below.
```

Comment:
How about if you add this sentence after the next sentence so it is rendered at the end of the bullet point?

Reply:
That was the intention. Thanks for the hint.